source/MaterialXRender/Handlers/TinyEXRImageLoader.cpp

```diff
@@ -0,0 +1,95 @@
+#include <MaterialXRender/Window/HardwarePlatform.h>
+
+#if defined(OSWin_) && defined(_WIN64)
+#define TINYEXR_USABLE
+#endif
+#if defined(OSMac_) || defined(OSLinux)
+#define TINYEXR_USABLE
+#endif
+
+#if defined(TINYEXR_USABLE)
+#define TINYEXR_IMPLEMENTATION
+#include <cstdlib>
+#include <cstdio>
+#include <limits>
+#include <stdio.h>
+#include <string.h>
+// Max may be defined in a macro so temporariy undef it.
+#ifdef max
+#define max_cache max
+#undef max
+#endif
+#include <MaterialXRender/External/tinyexr/tinyexr.h>
+#ifdef max_cache 
+#define max max_cache
+#endif
+#endif
+
+#include <MaterialXRender/Handlers/TinyEXRImageLoader.h>
+
+namespace MaterialX
+{
+bool TinyEXRImageLoader::supportsExtension(const std::string extension)
```

Comment:
File got renamed and class derivation changes but only new thing is supportsExtension() method to check which codecs are supported.